recipes/imagemagick/all/conanfile.py

```diff
@@ -100,336 +145,259 @@ def configure(self):
         if self.options.shared:
             del self.options.fPIC
 
+        if microsoft.is_msvc(self):
+            self.options.with_openmp = False
+        if self.settings.os != "Windows":
+            self.options.with_gdi32 = False
+
+        if self.settings.os == "Windows":
+            self.win_bash = True
```

Comment:
I think I've been seeing this set under the `build_requirements` method. It's pertinent to the build system and not the host system.